gradio/demo.py

```diff
@@ -0,0 +1,274 @@
+import collections
+import os
+import tempfile
+
+from matplotlib import gridspec
+from matplotlib import pyplot as plt
+import numpy as np
+from PIL import Image
+import urllib
+
+import tensorflow as tf
+import gradio as gr
+
+from subprocess import call
+import sys
+
+import requests
+
+url1 = 'https://cdn.pixabay.com/photo/2021/01/28/18/21/beach-5958718_1280.jpg'
+r = requests.get(url1, allow_redirects=True)
+open("town.jpg", 'wb').write(r.content)
+
+url2 = 'https://cdn.pixabay.com/photo/2014/05/20/21/20/bird-349026_1280.jpg'
+r = requests.get(url2, allow_redirects=True)
+open("bird.jpg", 'wb').write(r.content)
```

Comment:
for some reason this image fails at inference (it shows "error") on the top right corner of output.

Reply:
btw any chance we can inspect the error message? that would help debugging

Reply:
errors are shown in logs here https://gradio.app/glogs/AK391/deeplab2 as for the error it could be related to oom?

Reply:
cool! maybe add this pointer somewhere so anyone who encountered issues might be able to check this?